src/tests/functional/plugin/shared/include/behavior/ov_executable_network/get_metric.hpp

```diff
@@ -186,6 +187,28 @@ TEST_P(OVClassExecutableNetworkGetMetricTest_ThrowsUnsupported, GetMetricThrow)
     ASSERT_THROW(compiled_model.get_property("unsupported_property"), ov::Exception);
 }
 
+TEST_P(OVClassExecutableNetworkGetMetricTest_EXEC_DEVICES, GetMetricNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    std::vector<std::string> expectedTargets = {CommonTestUtils::DEVICE_CPU};
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device);
+
+    std::vector<std::string> exeTargets;
+    OV_ASSERT_NO_THROW(exeTargets = compiled_model.get_property(ov::execution_devices));
+
+    ASSERT_EQ(expectedTargets, exeTargets);
+}
+
+TEST_P(OVClassExecutableNetworkGetMetricTest_EXEC_DEVICES_GPU, GetMetricNoThrow) {
```

Comment:
better to parameterize these 2 tests, the test body are quite similar

Reply:
Parameterized these 2 tests.